src/version.c

```diff
@@ -64,18 +64,20 @@ init_longVersion(void)
 #else
 	char *date_time = __DATE__ " " __TIME__;
 #endif
-	char *msg = _("%s (%s, compiled %s)");
+	int hp = highest_patch();
+	char *msg = _("%s.%d (%s, compiled %s)");
 	size_t len = strlen(msg)
 		    + strlen(VIM_VERSION_LONG_ONLY)
 		    + strlen(VIM_VERSION_DATE_ONLY)
+		    + hp
```

Comment:
This seems wrong. `len` is the length of the string, which should be a fixed value rather that a runtime value. E.g. `msg_outnum` in message.c just uses 20.

Reply:
Oops, yes; fixed it.